src/index.js

```diff
@@ -4,11 +4,13 @@ import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
+  <script src="https://cdn.jsdelivr.net/npm/big.js@6.0.0/big.min.js" />;
```

Comment:
Apart from being a bad practice to add scripts this way, I don't think you need to import big.js this way since you already added this package to the package.json file.